administracion/views.py

```diff
@@ -5,26 +5,28 @@
 
 def admin_main_dashboard(request):
```

Comment:
this, and all the rest of the views should have the decorator @user_passes_test and @login_required